src/main/java/Duke.java

```diff
@@ -1,10 +1,282 @@
+                    System.out.println(LINE + invalid + LINE);
+                }
+            }
+
+            else if (isDelete(firstWord)  && wordLength <= 1) { //complete Tasks
+                invalid.setNotDelete();
+                System.out.println(LINE + invalid + LINE);
+            }
+
+            else if (!isBye(userInput) && !isList(userInput)) { //Task words
+                if (isTodo(firstWord) && wordLength == 1) { //Todo is empty
+                    invalid.setTodoEmpty();
+                    isInvalid = 1;
+                }
+                else if (isTodo(firstWord) && wordLength > 1){ //Todo is not empty
+                    Task.add(count, new Todo(userInput));
+                    count++;
+                }
+
+                else if (isDeadline(firstWord) && wordLength == 1 ) { //Deadline is empty
+                    invalid.setDeadlineEmpty();
+                    isInvalid = 1;
+                }
+
+                else if (isDeadline(firstWord) && wordLength > 1 ) { //Deadline is not empty
+                    if (userInput.contains("/")) { //Deadline Task has a date
+                        date = userInput.substring(userInput.lastIndexOf("/"));
+                        Task.add(count, new Deadline(userInput, date));
+                        count++;
+                    }
+                    else { //Deadline Task has no date
+                        invalid.setDeadlineNoDate();
+                        isInvalid = 1;
+                    }
+                }
+
+                else if (isEvent(firstWord) && wordLength == 1 ) { //Event task
+                    invalid.setEventEmpty();
+                    isInvalid = 1;
+                }
+
+                else if (isEvent(firstWord) && wordLength > 1 ) { //Event task
+                    if (userInput.contains("/")) { //Event Task has a date
+                        date = userInput.substring(userInput.lastIndexOf("/"));
+                        Task.add(count, new Event(userInput, date));
+                        count++;
+                    }
+                    else { //Deadline Task has no date
+                        invalid.setEventNoDate();
+                        isInvalid = 1;
+                    }
+                }
+
+                else { //Other invalid commands
+                    invalid.setInvalidCommand();
+                    isInvalid = 1;
+                }
+
+                switch(isInvalid) { //handles errors
+                    case 0: System.out.println(LINE + "Ok! I've added this " + firstWord.toLowerCase() + ":\n" +
+                            Task.get(count - 1) + "\n" + "Now you have " + count + (count == 1 ? " task":" tasks") + " in the list.\n" +
+                            "Type 'list' to save / view your tasks!\n" + LINE);
+                            break;
+                    case 1: System.out.println(LINE + invalid + LINE);
+                            break;
+                }
```

Comment:
Avoid indenting case clauses. You can configure the switch case format in the IDE.